### DISCUSSION TITLE
Need help with lat/long ref

### DISCUSSION THREAD

**Discussion by extellior** [Exiv2/exiv2#2157] [Q&A] [answered]
My bad on this post.  The moderator can delete it.

**Comment by clanmills** [comment, marked as answer]
GPSLatitudeRef and GPSLongitudeRef are null terminated ASCII.  So, two bytes. 

```
954 rmills@rmillsm1:~ $ exiv2 -pv -g Latitude/i -g Long/i ~/Stonehenge.jpg 
0x0001 GPSInfo      GPSLatitudeRef              Ascii       2  N
0x0002 GPSInfo      GPSLatitude                 Rational    3  51/1 106969/10000 0/1
0x0003 GPSInfo      GPSLongitudeRef             Ascii       2  W
0x0004 GPSInfo      GPSLongitude                Rational    3  1/1 495984/10000 0/1
955 rmills@rmillsm1:~ $ 
```
This is documented in Exif specification CIPA DC-008-2019 starting at page 72.  GPSInfo tags have been in Exif since I started to work on Exiv2 in 2008.  So, you'll find them documented in almost any version of the Exif spec.

I haven't worked with VB/C# since 2013.  I don't recognise the pattern of your code.  Are you using Exiv2 or another DLL to read the metadata in your image?
